README.md

```diff
@@ -143,6 +143,7 @@ Parameter | Type | User Property | Required | Description
 --- | --- | --- | --- | ---
 `<chartDirectory>` | string | helm.chartDirectory | true | root directory of your charts
 `<chartVersion>` | string | helm.chartVersion | true | Version of the charts. The version have to be in the [SEMVER-Format](https://semver.org/), required by helm.
+`<appVersion>` | string | helm.appVersion | true | Version of the Application. The version have to be in the [SEMVER-Format](https://semver.org/), required by helm.
```

Comment:
appVersion is an optional field which not has to be SemVer.
https://docs.helm.sh/developing_charts/#the-chart-yaml-file

Reply:
Fixed